Libraries/LibWeb/IndexedDB/IDBObjectStore.cpp

```diff
@@ -45,6 +45,16 @@ void IDBObjectStore::visit_edges(Visitor& visitor)
     visitor.visit(m_indexes);
 }
 
+GC::Ptr<Index> IDBObjectStore::index_with_name(String const& name) const
```

Comment:
Yet another method that makes me ask "why not use a HashMap of name to index?"

Reply:
The main reason is that it was slightly akward.. Indexes can be renamed, so it would need to be "re-keyed" whenever that happened